### ISSUE TITLE
Accordian: use `hidden-until-found` internally to support browser native "Find in Page"

### ISSUE THREAD

**Issue by ajitzero** [ng-primitives/ng-primitives#82]
### Feature Type

Enhancement

### Feature Description

- Reference: https://developer.chrome.com/docs/css-ui/hidden-until-found
- Caveat: This is for collapsible sections that exist in the DOM, so not applicable for sections hidden with `ngIf/@if`

**Current Behavior:**

Try searching for the term "open source" on this page: https://angularprimitives.com/primitives/accordion.

**Expected Behavior:**

Browser-native search should work for hidden/collapsed sections.

**Comment by ashley-hunter** [comment]
Thank you very much for the suggestion! I actually came across this same article a week or two ago myself. I haven't implemented it yet because the feature is still marked as `experimental` and has limited browser support:

![image](https://github.com/user-attachments/assets/80802b9d-5183-40fe-bf8f-82f3f2e9f268)
https://caniuse.com/?search=hidden%3Duntil-found%20

But I will leave this issue open as this is definitely something I would like to do as soon as possible. Great suggestion, thank you!

**Comment by ajitzero** [comment]
Yeah, that makes sense! I saw that it has been implemented in React Aria as a part of the Disclosure component so I thought it would be great to include here as well. Plus, it is a progressive enhancement and doesn't impact browsers that don't support it.

**Comment by ashley-hunter** [comment]
Ah interesting, if that's the case then we could probably add this for browsers that support it! Thank you 😊
